tools/dcos_login.py

```diff
@@ -203,16 +143,100 @@ def ignore_empty(envvar, default):
     dcos_login_username = ignore_empty("DCOS_LOGIN_USERNAME", __CLI_LOGIN_EE_USERNAME)
     dcos_login_password = ignore_empty("DCOS_LOGIN_PASSWORD", __CLI_LOGIN_EE_PASSWORD)
     dcos_enterprise = ignore_empty("DCOS_ENTERPRISE", "true").lower() == "true"
-    dcos_acs_token = os.environ.get("DCOS_ACS_TOKEN")
-    if not dcos_acs_token:
-        dcos_acs_token = login(
-            dcosurl=cluster_url,
-            username=dcos_login_username,
-            password=dcos_login_password,
-            is_enterprise=dcos_enterprise,
-        )
-    configure_cli(dcosurl=cluster_url, token=dcos_acs_token)
+    if is_old_cli:
+        # TODO(takirala) : We can remove this code path few months after 1.12 GA
+        _run_cmd("dcos config set core.dcos_url {}".format(cluster_url))
+        _run_cmd("dcos config set core.ssl_verify {}".format(False))
+        if dcos_enterprise:
+            _run_cmd(
+                "dcos auth login --username {} --password {}".format(
+                    dcos_login_username,
+                    dcos_login_password
+                )
+            )
+        else:
+            _run_cmd("dcos config set core.dcos_acs_token {}".format(
+                os.environ.get("DCOS_ACS_TOKEN", login(
+                    dcosurl=cluster_url,
+                    username=dcos_login_username,
+                    password=dcos_login_password,
+                    is_enterprise=dcos_enterprise,
+                ))),
+                check=True
+            )
+    else:
+        if dcos_enterprise:
+            _run_cmd(
+                "dcos cluster setup {} --username {} --password {} --insecure".format(
+                    cluster_url,
+                    dcos_login_username,
+                    dcos_login_password
+                ),
+                check=True
+            )
+        else:
+            _run_cmd(
+                "env DCOS_ACS_TOKEN={} dcos cluster setup {} --insecure".format(
```

Comment:
@bamarni is this the correct usage of the command for 1.10+ open clusters with the latest cli ?

Reply:
Yep :+1: